Engine/AppInstance.h

```diff
@@ -193,22 +193,22 @@ GCC_DIAG_SUGGEST_OVERRIDE_ON
     virtual void warningDialog(const std::string & title, const std::string & message, bool* stopAsking, bool useHtml) const;
     virtual void informationDialog(const std::string & title, const std::string & message, bool useHtml) const;
     virtual void informationDialog(const std::string & title, const std::string & message, bool* stopAsking, bool useHtml) const;
-    virtual NATRON_ENUM::StandardButtonEnum questionDialog(const std::string & title,
+    virtual Natron::StandardButtonEnum questionDialog(const std::string & title,
                                               const std::string & message,
                                               bool useHtml,
-                                              NATRON_ENUM::StandardButtons buttons =
-                                                  NATRON_ENUM::StandardButtons(NATRON_ENUM::eStandardButtonYes | NATRON_ENUM::eStandardButtonNo),
-                                              NATRON_ENUM::StandardButtonEnum defaultButton = NATRON_ENUM::eStandardButtonNoButton) const WARN_UNUSED_RETURN;
+                                              Natron::StandardButtons buttons =
+                                                  Natron::StandardButtons(Natron::StandardButtonEnum::eStandardButtonYes | Natron::StandardButtonEnum::eStandardButtonNo),
```

Comment:
Do we still need to explicitly specify the namespace here and elsewhere NATRON_ENUM was used? I don't believe most(all?) are needed for compiling.

Reply:
This is needed as is, both namespace and enum specified, or else generation yields wrong results. Even Qt does kinda the same.